pkg/cmdutil/util.go

```diff
@@ -0,0 +1,31 @@
+package cmdutil
```

Comment:
Maybe a good name for this file would be `json` something?

Reply:
Renamed this file to `mapConversion.go` for clarity, and since the only thing this file contains now is the `StringMapToInterfaceMap` function. 3638380